src/utill.rs

```
@@ -204,8 +201,9 @@ pub fn setup_maker_logger(filter: LevelFilter) {
             )
             .build(Root::builder().appender("stdout").build(filter))
             .unwrap();
-
-        log4rs::init_config(config).unwrap();
+        if let Err(e) = log4rs::init_config(config) {
+            eprintln!("Failed to initialize logger: {}", e);
+        }
```

don't suppress the error. unwrap.